Sunrise/AppDelegate.swift

```diff
@@ -63,6 +63,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
                 AppRouting.switchToProductDetails(for: sku)
                 return true
 
+            // Orders (e.g https://demo.commercetools.com/en/user/orders/87896195?)
```

Comment:
It actually does - that is what I get on demo.commercetools.com when viewing an order.